src/events/api/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from events import api
+
+app_name = 'app'
+
+urlpatterns = [
+    path('talkproposal/<int:pk>', api.TalkDetailAPIView.as_view()),
```

Comment:
I prefer use talk/<int:pk>, instead of talkproposal(talkproposal include not accepted talk).

Reply:
reasonable